File: Pohon Natal/C#/pohon-natal-i.cs
using System;
class pohonNatal {
  static void Main() {
    Console.Write("Masukan Tinggi Pohon : ");
    int x = int.Parse(Console.ReadLine());
    for(int i = 1; i <= x; i++){
        for(int j = x; j >= i; j--){
            Console.Write(" ");
        }
        
        for(int l = 2; l <= i; l++){
            Console.Write("*");
        }
        
        for(int k = 1; k <= i; k++){
                Console.Write("*");
            }
            
        Console.Write("\n");
            
        if(i==x){
            for(int m = 0; m < 3; m++){
                for(int o = 1; o < i; o++){
                Console.Write(" ");
                }
                
                for(int n = 0; n <= 0; n++){
                Console.Write("===");
                }
                
            Console.Write("\n");
            }
        }
    }
  }
}

File: Pohon Natal/PHP/pohon-natal-i.php
<?php

$x = readline("Masukan Tinggi Pohon : ");
for($i = 1; $i <= $x; $i++){
    for($j = $x; $j >= $i; $j--){
        echo " ";
    }
    
    for($l = 2; $l <= $i; $l++){
        echo "*";
    }
    
    for($k = 1; $k <= $i; $k++){
            echo "*";
        }
        
    echo "\n";
        
    if($i==$x){
        for($m = 0; $m < 3; $m++){
            for($o = 1; $o < $i; $o++){
            echo " ";
            }
            
            for($n = 0; $n <= 0; $n++){
            echo "===";
            }
        echo "\n";
        }
    }
}
